modules/support.lua

```diff
@@ -163,17 +175,8 @@ selectPreferred = function(searchMethod)
 end
 
 scrollList = function()
-	local touchActions = {
-		{ action = "touchDown", target = game.SUPPORT_SWIPE_START_CLICK },
-		{ action = "touchMove", target = game.SUPPORT_SWIPE_END_CLICK },
-		{ action =      "wait", target = 0.4 },
-		{ action =   "touchUp", target = game.SUPPORT_SWIPE_END_CLICK },
-		{ action =      "wait", target = 0.5 } -- leaving some room for animations to finish
-	}
-
-	-- the movement has to be as accurate as possible
-	setManualTouchParameter(5, 5)
-	manualTouch(touchActions)
+	swipe(game.SUPPORT_SWIPE_START_CLICK,game.SUPPORT_SWIPE_END_CLICK,.5)
```

Comment:
I also tried using `swipe` when I originally wrote this function, but unfortunately it doesn't work for all devices, so I used `manualTouch` for better compatibility. Here's what happens on my phone: https://www.youtube.com/watch?v=X4eKXNaOyjk

Reply:
Alright, code has been reverted back to the Manual Touch function.